cdc-quickstart-kafka-connect/prometheus.yml

```diff
@@ -0,0 +1,15 @@
+global:
+ scrape_interval: 3s
+ evaluation_interval: 3s
+scrape_configs:
+
+ - job_name: 'kafka-connect-host'
+   static_configs:
+   - targets: ['kafka-connect:9876']
+
+ - job_name: 'yugabyted'
+   metrics_path: '/prometheus-metrics'
+   static_configs:
+   - targets: ['172.165.20.83:9000']
```

Comment:
Is this IP always the same?

Reply:
No it was my dev server's IP. I have removed hard coded part now